### PR TITLE
Added dev deployment to the README (CU-86dqu45n1)

### PR DESCRIPTION
**Pull request by Darrickg**
Dev deployment added to the README

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-86dqu45n1 Add how to deploy to dev in Buttons README](https://app.clickup.com/t/86dqu45n1)

File: README.md [hunk 1]
```diff
@@ -96,6 +96,36 @@
...
+## 3. Verify
+
+1. open the production dashboard and confirm that everything appears to be working normally
+
+1. Verify that the logs look reasonable
+
+   1. on the production Buttons server (access using ssh):
+
+      1. run `pm2 logs`
+
```
Comment (johnny-brave) [inline README.md:128]: "production" being used here when it should be "dev". Although, i think the whole verify step can be removed more dev deployment

Comment (Darrickg) [reply README.md]: fixed in newest push

Comment (Darrickg) [reply README.md]: i removed the entire verify section since it's not necessary

**Comment by Darrickg** [comment]
pulled from main my changes for the npm stuff, fixed travis

**Review by micahdbak — APPROVED** [review]
Looks good! Concise

**Pull request merged**